ogc_ws/src/sbd/launch/sbd_gndtest.launch

```diff
@@ -59,7 +59,11 @@
     </node>
 
     <!-- rqt node -->
+<<<<<<< HEAD
```

Comment:
Typo? Looks like the merge conflicts weren't completed fully

Reply:
Yep sorry fixed it in the newest commit